examples/PNG Images/Flash_transparent_PNG/Flash_transparent_PNG.ino

```diff
@@ -88,3 +88,23 @@ void loop()
   // Fill screen with a random colour at random intervals
   if (random(100) < 20) tft.fillScreen(random(0x10000));
 }
+
+
+
+//=========================================v==========================================
+//                                      pngDraw
+//====================================================================================
+// This next function will be called during decoding of the png file to
+// render each image line to the TFT.  If you use a different TFT library
+// you will need to adapt this function to suit.
+// Callback function to draw pixels to the display
+void pngDraw(PNGDRAW *pDraw)
+{
+  uint16_t lineBuffer[MAX_IMAGE_WIDTH];
+  uint8_t pMask[40];
+
+  png.getLineAsRGB565(pDraw, lineBuffer, PNG_RGB565_BIG_ENDIAN, 0xffffffff);
+  if (png.getAlphaMask(pDraw, pMask, 200)) { // if any pixels are opaque, draw them
```

Comment:
I'm not sure why using 255 for `ucThreshold` gives me jagged edges on my example image. I also get bad results with very small values (0).